.github/CODEOWNERS

```
@@ -350,6 +350,7 @@
 /test/kitchen/test/integration/win-reinstall-option/ @DataDog/windows-agent
 /test/kitchen/test/integration/win-repair/ @DataDog/windows-agent
 /test/kitchen/test/integration/win-user/ @DataDog/windows-agent
+/test/new-e2e                           @DataDog/agent-e2e-testing
```

```suggestion
/test/new-e2e/                          @DataDog/agent-e2e-testing
```

for consistency with the rest of the file